mail_activity_board/views/mail_activity_view.xml

```diff
@@ -124,119 +50,22 @@
-                                                <span t-attf-class="text-danger">
-                                                    <field
-                                                        name="calendar_event_id_start"
-                                                    />
-                                                </span>
-                                            </t>
-                                        </t>
-                                        <t t-else="">
-                                            <span>
-                                                <i title="date" class="fa fa-clock-o" />
-                                            </span>
-                                            <t
-                                                t-if="record.activity_category.raw_value!='meeting'"
-                                            >
-                                                <span>
-                                                    <field name="date_deadline" />
-                                                </span>
-                                            </t>
-                                            <t t-else="">
-                                                <field name="calendar_event_id_start" />
-                                            </t>
-                                        </t>
-                                    </div>
-                                    <div class="oe_kanban_bottom_right">
-                                        <field
-                                            name="user_id"
-                                            widget="many2one_avatar_user"
-                                        />
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </t>
-                </templates>
-            </kanban>
-        </field>
-    </record>
-
     <!-- SEARCH VIEW -->
     <record id="mail_activity_view_search" model="ir.ui.view">
         <field name="name">mail.activity.boards.view.search</field>
         <field name="model">mail.activity</field>
         <field name="inherit_id" ref="mail.mail_activity_view_search" />
-        <field name="priority" eval="2" />
-        <field name="mode">primary</field>
+        <field name="mode">extension</field>
         <field name="arch" type="xml">
             <xpath expr='//field[@name="res_model"]' position='before'>
                 <field name="user_id" />
                 <field name="res_name" string="Origin" />
             </xpath>
 
-            <xpath expr='//filter[@name="filter_date_deadline_past"]' position='before'>
+            <xpath
+                expr='//filter[@name="filter_date_deadline_future"]'
+                position='before'
+            >
                 <filter
                     string="Act. next month"
                     name="activities_month"
```

Comment:
I am not a big fan of "Act. Next month" and "Act. Next 6 Months" filter because they are very specific. Maybe we should remove them as well?

Reply:
Agree! You can replace with something like

```
<filter name="date_deadline" date="date_deadline" start_year="0" start_month="-1" end_year="2" end_month="2" />
```

<img width="204" height="463" alt="Image" src="https://github.com/user-attachments/assets/3abaf6cc-f48a-4e0f-83af-998a86d02a77" />